superset/charts/commands/importers/v1/__init__.py

```diff
@@ -96,10 +96,5 @@ def _import(
                     }
                 )
                 config["params"].update({"datasource": dataset.uid})
-                if config["query_context"]:
```

Comment:
We need to delete query_context if it exists:

```python
# fix old exports before PR
if config["query_context"]:
    config["query_context"] = None
```

Reply:
Would it be better if this was 
```python
if config["query_context"]:
   del config["query_context"]
```